server/src/controllers/Fields.ts

```diff
@@ -0,0 +1,8 @@
+
+export type Fields = {
```

Comment:
pega tudo que for types e cria um arquivo dentro de `/src` chamado `types.ts`, taca tudo lá